lib/praxis/action_definition.rb

```diff
@@ -31,7 +31,7 @@ def self.decorate_docs(&callback)
     end
 
     def initialize(name, resource_definition, **opts, &block)
-      @name = name
+      @name = name.to_sym
```

Comment:
Yeah. I saw the spec that does it, surprised that actually worked. I'd say we fix that spec and not coerce the name coming in, just document that it should always be a symbol? 

It's not something user's instantiate directly, either.